src/pcapi/app.py

```diff
@@ -33,6 +34,7 @@ def install_login_manager() -> None:
     install_admin_views(admin, db.session)
     install_routes(app)
 
+    app.register_blueprint(eac_v1, url_prefix="/eac/v1")
```

Comment:
```suggestion
    app.register_blueprint(eac_v1, url_prefix="/adage/v1")
```

Ou est-ce que tu veux regrouper les end-points pour adage et educonnect ?

Reply:
C'est pas faux, un partenaire peut toujours changer de nom en plus
Mais là je n'ai pas d'idées sur un terme plus générique :thinking: 
`/collective` ? (je ne suis pas convaincu hein :P )